### PR TITLE
scss & css with modules

### LINKED ISSUE #19: Scss & css modules
Add a section about how to use scss and css modules with Falcon

### PR DESCRIPTION
**Pull request by tutok**
resolve #19

### CODE REVIEW

File: docs/falcon-client/basics.md [hunk 1]
```diff
@@ -144,23 +146,33 @@ To change page title or add any kind of meta tag (e.g. [Open Graph Protocol](htt
...
 `googleTagManager: object`
 
 - `id: string`: (default `null`) Google Tag Manager ID
 
-## Internal Server Error page
+## Development
+
+### Styles
+
+`falcon-client` provide out of the box support for `css` and `scss`, and you should write them in `*css` and `*.scss` files accordingly.
```
Comment (Borales) [inline docs/falcon-client/basics.md]:
```suggestion
`falcon-client` provides out of the box support for `css` and `scss`, and you should write them in `*.css` and `*.scss` files accordingly.
```

File: docs/falcon-client/basics.md [hunk 2]
```diff
@@ -144,23 +146,33 @@ To change page title or add any kind of meta tag (e.g. [Open Graph Protocol](htt
...
+
+However, css modules convention is also supported [see the details](https://github.com/css-modules/css-modules). To make it use you need to add `module` prefix to file extension. For example, vanilla css with modules should be located in `*.module.css`, and scss with modules in `*.module.scss`.
```
Comment (Borales) [inline docs/falcon-client/basics.md]:
```suggestion
However, css modules convention is also supported [see the details](https://github.com/css-modules/css-modules). To make it work you need to add `module` prefix to the file extension. For example, vanilla css with modules should be located in `*.module.css`, and scss with modules in `*.module.scss`.
```

File: docs/falcon-client/basics.md [hunk 3]
```diff
@@ -179,17 +191,17 @@ you can configure Google Tag Manager via `config` property in `falcon-client.con
 example unit test with `FalconClientMock` :
 
 ```jsx
-import { FalconClientMock } from '@deity/falcon-client/test-utils';
```
Comment (Borales) [inline docs/falcon-client/basics.md]: @tutok what was wrong with single quotes? (our code uses single quotes)

Comment (tutok) [reply docs/falcon-client/basics.md]: nothing, #21

**Pull request merged**